x-pack/plugins/apm/server/routes/storage_explorer/get_summary_statistics.ts

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ProcessorEvent } from '@kbn/observability-plugin/common';
+import { QueryDslQueryContainer } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import {
+  termQuery,
+  kqlQuery,
+  rangeQuery,
+} from '@kbn/observability-plugin/server';
+import {
+  getTotalIndicesStats,
+  getEstimatedSizeForDocumentsInIndex,
+} from './indices_stats_helpers';
+import { Setup } from '../../lib/helpers/setup_request';
+import { ApmPluginRequestHandlerContext } from '../typings';
+import {
+  IndexLifecyclePhaseSelectOption,
+  indexLifeCyclePhaseToDataTier,
+} from '../../../common/storage_explorer_types';
+import { RandomSampler } from '../../lib/helpers/get_random_sampler';
+import {
+  SERVICE_NAME,
+  TIER,
+  INDEX,
+  TRANSACTION_ROOT,
+  PARENT_ID,
+} from '../../../common/elasticsearch_fieldnames';
+import { environmentQuery } from '../../../common/utils/environment_query';
+import {
+  getDocumentTypeFilterForTransactions,
+  getProcessorEventForTransactions,
+} from '../../lib/helpers/transactions';
+import { calculateThroughputWithRange } from '../../lib/helpers/calculate_throughput';
+
+export async function getTracesPerMinute({
+  setup,
+  indexLifecyclePhase,
+  start,
+  end,
+  environment,
+  kuery,
+  searchAggregatedTransactions,
+}: {
+  setup: Setup;
+  indexLifecyclePhase: IndexLifecyclePhaseSelectOption;
+  start: number;
+  end: number;
+  environment: string;
+  kuery: string;
+  searchAggregatedTransactions: boolean;
+}) {
+  const { apmEventClient } = setup;
+
+  const response = await apmEventClient.search('get_traces_per_minute', {
+    apm: {
+      events: [getProcessorEventForTransactions(searchAggregatedTransactions)],
+    },
+    body: {
+      size: 0,
+      track_total_hits: false,
+      query: {
+        bool: {
+          filter: [
+            ...getDocumentTypeFilterForTransactions(
+              searchAggregatedTransactions
+            ),
+            ...environmentQuery(environment),
+            ...kqlQuery(kuery),
+            ...rangeQuery(start, end),
+            ...(indexLifecyclePhase !== IndexLifecyclePhaseSelectOption.All
+              ? termQuery(
+                  TIER,
+                  indexLifeCyclePhaseToDataTier[indexLifecyclePhase]
+                )
+              : []),
+            ...(searchAggregatedTransactions
+              ? [
+                  {
+                    term: {
+                      [TRANSACTION_ROOT]: true,
+                    },
+                  },
+                ]
+              : []),
```

Comment:
Why do we only want to limit this to root transactions for aggregated transactions?

Reply:
Ahh, I see you have the other case covered below

```ts
must_not: {
                  exists: { field: PARENT_ID },
                },
```